kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -138,6 +138,14 @@ window.ExerciseView = Backbone.View.extend({
 
         // Catch the "next question" button click event -- needs to be explicit (not in "events")
         this.$("#next-question-button").click(this.next_question_clicked);
+        
+        // Workaround so that pressing enter on perseus exercises submits the answer 
```

Comment:
Maybe I'm missing something, but there doesn't appear to be a Backbone event corresponding to "keydown": http://backbonejs.org/#Events-catalog